dataprofiler/profilers/profile_builder.py

```diff
@@ -421,27 +418,16 @@ def __init__(self, data, samples_per_update=None, min_true_samples=0,
         self._min_true_samples = min_true_samples
         self._profile = dict()
 
+        # matches structured data profile
+        # TODO: allow set via options
+        self._sampling_ratio = 0.2
+        self._min_sample_size = 5000
```

Comment:
Isn't this set by default already when you create the class?

Reply:
No, these are new to this class.